core/src/core-plugin-definitions.ts

```diff
@@ -1379,6 +1379,10 @@ export interface StatusBarPlugin extends Plugin {
    *  Hide the status bar
    */
   hide(): Promise<void>;
+  /**
+   *  Get info about the current state of the status bar
+   */
+  getInfo(): Promise<void>;
```

Comment:
Should be StatusBarInfoResult instead of void.

Will give it a try tomorrow

Reply:
Fixed. Cool thanks 👍